web/packages/teleterm/src/ui/initUi.tsx

```diff
@@ -53,9 +56,42 @@ function notifyAboutStoredConfigErrors(
   if (errors) {
     notificationsService.notifyError({
       title: 'Encountered errors in config file',
-      description: errors
-        .map(error => `${error.path[0]}: ${error.message}`)
-        .join('\n'),
+      description: (
+        <span>
+          <ErrorsRenderer
+            errors={errors.map(e => `${e.path[0].toString()}: ${e.message}`)}
+          />
+          {/**TODO(gzdunek): point to the properer section */}
+          <Link
+            href={
+              'https://goteleport.com/docs/connect-your-client/teleport-connect/'
+            }
+            target="_blank"
+          >
+            See documentation for the app config.
```

Comment:
I'd typically exclude punctuation from or make "the documentation for the app config" the text of the link.

Reply:
Done